src/context/body.test.ts

```diff
@@ -7,3 +8,12 @@ test('sets a given body value without implicit "Content-Type" header', async ()
   expect(result).toHaveProperty('body', 'Lorem ipsum')
   expect(result.headers.get('content-type')).toBeNull()
 })
+
+test('does not stringify raw body twice if content is string and "Content-Type" header is "json"', async () => {
+  const result = await response(
+    set('Content-Type', 'application/json'),
+    body(JSON.stringify('some text')),
+  )
+
+  expect(result).toHaveProperty('body', `"some text"`)
+})
```

Comment:
we need this test anymore?

Reply:
We still need to ensure that the response body doesn't get stringified twice, no matter the header.